recipes/credentials.rb

```diff
@@ -0,0 +1,26 @@
+#
+# Cookbook Name:: osl-jenkins
+# Recipe:: default
+#
+# Copyright 2014, Oregon State University
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+jenkins = encrypted_data_bag_item('ssh-keys', 'osl-jenkins')
```

Comment:
@armiller I did a bit of digging to see why the resource wasn't giving a meaningful error message. If that resource fails, it raises a "404 Not Found" error. The problem is there are two errors that could arise that I believe Chef doesn't check:

 1) The data bag doesn't exist
 2) The data bag item doesn't exist

Chef merely does an HTTP(s) request to the Chef server for the databag and doesn't wrap the error in anything meaningful: https://github.com/chef/chef/blob/master/lib/chef/encrypted_data_bag_item.rb#L123 -> https://github.com/chef/chef/blob/master/lib/chef/data_bag_item.rb#L152

I will add the rescue block, but it will be a bit ambiguous.